spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  let!(:user) do
+    User.create(name: 'Jhader', keywords: ['Testing'], email: 'testing2@test.com', password: 'qwerty123')
+  end
+
+  let(:user_session) do
+    session[:user_id] = user.id
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      user_session
+      get :show
+      json_response = JSON.parse(response.body).first[1]
```

Comment:
You can define helper methods for things you execute on your tests.

Generally they lie under `spec/support/` and are imported and included to the rspec testsuite on the `spec/rails_helper.rb`.